components/Common/AvatarGroup/index.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import { githubProfileAvatarUrl } from '@/util/avatars';
+
+import AvatarGroup from './';
+
+type Story = StoryObj<typeof AvatarGroup>;
+type Meta = MetaObj<typeof AvatarGroup>;
+
+const names = [
+  'ovflowd',
+  'bmuenzenmeyer',
+  'AugustinMauroy',
+  'HinataKah0',
+  'Harkunwar',
+  'rodion-arr',
+  'mikeesto',
+  'bnb',
+  'benhalverson',
+  'aymen94',
+  'shanpriyan',
+  'Wai-Dung',
+  'manishprivet',
+  'araujogui',
+];
+
+// we are using github avatars for this story
+// an link to other image can also be used
+const defaultProps = {
+  avatars: names.map(name => ({
+    src: githubProfileAvatarUrl(name),
+    alt: name,
+  })),
+};
+
+// add falback text to avatars we are hardcoding link because github api add image for 404 user avatar
+defaultProps.avatars.push({
```

Comment:
I don't think .push should be called here. If you want to add this to avatars on defaultProps... You can simply:

```tsx
const unknownAvatar = [{
  src: 'https://avatars.githubusercontent.com/u/',
  alt: 'unknown-avatar'
});

const defaultProps = {
  avatars: [unknownAvatar, ...avatars.map(....)],
};
```